diff --git a/README.md b/README.md
index 770f663..55e1b91 100644
--- a/README.md
+++ b/README.md
@@ -228,15 +228,15 @@ cat examples/pi/pi-mpich.yaml
 
 | Metric name | Metric type | Description | Labels |
 | ----------- | ----------- | ----------- | ------ |
-|mpi\_operator\_jobs\_created\_total | Counter  | Counts number of MPI jobs created | |
-|mpi\_operator\_jobs\_successful\_total | Counter  | Counts number of MPI jobs successful | |
-|mpi\_operator\_jobs\_failed\_total | Counter  | Counts number of MPI jobs failed| |
-|mpi\_operator\_job\_info | Gauge | Information about MPIJob | `launcher`=&lt;launcher-pod-name&gt; <br> `namespace`=&lt;job-namespace&gt; |
+|group\_operator\_jobs\_created\_total | Counter  | Counts number of Group jobs created | |
+|group\_operator\_jobs\_successful\_total | Counter  | Counts number of Group jobs successful | |
+|group\_operator\_jobs\_failed\_total | Counter  | Counts number of Group jobs failed| |
+|group\_operator\_job\_info | Gauge | Information about GroupJob | `launcher`=&lt;launcher-pod-name&gt; <br> `namespace`=&lt;job-namespace&gt; |
 
 ### Join Metrics
 
 With [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics), one can join metrics by labels.
-For example `kube_pod_info * on(pod,namespace) group_left label_replace(mpi_operator_job_infos, "pod", "$0", "launcher", ".*")`
+For example `kube_pod_info * on(pod,namespace) group_left label_replace(group_operator_job_infos, "pod", "$0", "launcher", ".*")`
 
 ## Docker Images
 
diff --git a/cmd/mpi-operator/app/server.go b/cmd/mpi-operator/app/server.go
index 0ff59cb..f1b3c7a 100644
--- a/cmd/mpi-operator/app/server.go
+++ b/cmd/mpi-operator/app/server.go
@@ -75,10 +75,10 @@ var (
 )
 
 var (
-	isLeader = promauto.NewGauge(prometheus.GaugeOpts{
-		Name: "mpi_operator_is_leader",
-		Help: "Is this client the leader of this mpi-operator client set?",
-	})
+       isLeader = promauto.NewGauge(prometheus.GaugeOpts{
+               Name: "group_operator_is_leader",
+               Help: "Is this client the leader of this group-operator client set?",
+       })
 )
 
 func Run(opt *options.ServerOption) error {
diff --git a/pkg/apis/kubeflow/v2beta1/constants.go b/pkg/apis/kubeflow/v2beta1/constants.go
index 3387176..34d8413 100644
--- a/pkg/apis/kubeflow/v2beta1/constants.go
+++ b/pkg/apis/kubeflow/v2beta1/constants.go
@@ -22,7 +22,7 @@ const (
 	// DefaultLauncherRestartPolicy is default RestartPolicy for Launcher Job.
 	DefaultLauncherRestartPolicy = RestartPolicyOnFailure
 	// OperatorName is the name of the operator used as value to the label common.OperatorLabelName
-	OperatorName = "mpi-operator"
+       OperatorName = "group-operator"
 )
 
 // merge from common.v1
@@ -30,17 +30,17 @@ const (
 const (
 
 	// ReplicaIndexLabel represents the label key for the replica-index, e.g. 0, 1, 2.. etc
-	ReplicaIndexLabel = "training.kubeflow.org/replica-index"
+       ReplicaIndexLabel = "training.cw.xyz/replica-index"
 
 	// ReplicaTypeLabel represents the label key for the replica-type, e.g. ps, worker etc.
-	ReplicaTypeLabel = "training.kubeflow.org/replica-type"
+       ReplicaTypeLabel = "training.cw.xyz/replica-type"
 
 	// OperatorNameLabel represents the label key for the operator name, e.g. tf-operator, mpi-operator, etc.
-	OperatorNameLabel = "training.kubeflow.org/operator-name"
+       OperatorNameLabel = "training.cw.xyz/operator-name"
 
 	// JobNameLabel represents the label key for the job name, the value is the job name.
-	JobNameLabel = "training.kubeflow.org/job-name"
+       JobNameLabel = "training.cw.xyz/job-name"
 
 	// JobRoleLabel represents the label key for the job role, e.g. master.
-	JobRoleLabel = "training.kubeflow.org/job-role"
+       JobRoleLabel = "training.cw.xyz/job-role"
 )
diff --git a/pkg/controller/mpi_job_controller.go b/pkg/controller/mpi_job_controller.go
index c00a98e..25046dc 100644
--- a/pkg/controller/mpi_job_controller.go
+++ b/pkg/controller/mpi_job_controller.go
@@ -117,22 +117,22 @@ const (
 )
 
 var (
-	mpiJobsCreatedCount = promauto.NewCounter(prometheus.CounterOpts{
-		Name: "mpi_operator_jobs_created_total",
-		Help: "Counts number of MPI jobs created",
-	})
-	mpiJobsSuccessCount = promauto.NewCounter(prometheus.CounterOpts{
-		Name: "mpi_operator_jobs_successful_total",
-		Help: "Counts number of MPI jobs successful",
-	})
-	mpiJobsFailureCount = promauto.NewCounter(prometheus.CounterOpts{
-		Name: "mpi_operator_jobs_failed_total",
-		Help: "Counts number of MPI jobs failed",
-	})
-	mpiJobInfoGauge = promauto.NewGaugeVec(prometheus.GaugeOpts{
-		Name: "mpi_operator_job_info",
-		Help: "Information about MPIJob",
-	}, []string{"launcher", "namespace"})
+       mpiJobsCreatedCount = promauto.NewCounter(prometheus.CounterOpts{
+               Name: "group_operator_jobs_created_total",
+               Help: "Counts number of Group jobs created",
+       })
+       mpiJobsSuccessCount = promauto.NewCounter(prometheus.CounterOpts{
+               Name: "group_operator_jobs_successful_total",
+               Help: "Counts number of Group jobs successful",
+       })
+       mpiJobsFailureCount = promauto.NewCounter(prometheus.CounterOpts{
+               Name: "group_operator_jobs_failed_total",
+               Help: "Counts number of Group jobs failed",
+       })
+       mpiJobInfoGauge = promauto.NewGaugeVec(prometheus.GaugeOpts{
+               Name: "group_operator_job_info",
+               Help: "Information about GroupJob",
+       }, []string{"launcher", "namespace"})
 
 	sshVolumeItems = []corev1.KeyToPath{
 		{
diff --git a/rebranding.patch b/rebranding.patch
index a4ab42e..40777f6 100644
--- a/rebranding.patch
+++ b/rebranding.patch
@@ -1,71 +0,0 @@
-diff --git a/README.md b/README.md
-index 5d710dd..770f663 100644
---- a/README.md
-+++ b/README.md
-@@ -1,9 +1,9 @@
--# MPI Operator
-+# Group Operator
- 
- [![Build Status](https://github.com/kubeflow/mpi-operator/workflows/build/badge.svg)](https://github.com/kubeflow/mpi-operator/actions?query=event%3Apush+branch%3Amaster)
- [![Docker Pulls](https://img.shields.io/docker/pulls/mpioperator/mpi-operator)](https://hub.docker.com/r/mpioperator/mpi-operator)
- 
--The MPI Operator makes it easy to run allreduce-style distributed training on Kubernetes. Please check out [this blog post](https://medium.com/kubeflow/introduction-to-kubeflow-mpi-operator-and-industry-adoption-296d5f2e6edc) for an introduction to MPI Operator and its industry adoption.
-+The Group Operator makes it easy to run allreduce-style distributed training on Kubernetes. Please check out [this blog post](https://medium.com/kubeflow/introduction-to-kubeflow-mpi-operator-and-industry-adoption-296d5f2e6edc) for an introduction to Group Operator and its industry adoption.
- 
- ## Installation
- 
-diff --git a/deploy/v2beta1/mpi-operator.yaml b/deploy/v2beta1/mpi-operator.yaml
-index caa0651..107461c 100644
---- a/deploy/v2beta1/mpi-operator.yaml
-+++ b/deploy/v2beta1/mpi-operator.yaml
-@@ -27,14 +27,14 @@ metadata:
-     app.kubernetes.io/component: mpijob
-     app.kubernetes.io/name: mpi-operator
-     kustomize.component: mpi-operator
--  name: mpijobs.kubeflow.org
-+  name: groupjobs.cw.xyz
- spec:
--  group: kubeflow.org
-+  group: cw.xyz
-   names:
--    kind: MPIJob
--    listKind: MPIJobList
--    plural: mpijobs
--    singular: mpijob
-+    kind: GroupJob
-+    listKind: GroupJobList
-+    plural: groupjobs
-+    singular: groupjob
-   scope: Namespaced
-   versions:
-   - name: v2beta1
-diff --git a/pkg/apis/kubeflow/v2beta1/doc.go b/pkg/apis/kubeflow/v2beta1/doc.go
-index 45eb00b..25ce134 100644
---- a/pkg/apis/kubeflow/v2beta1/doc.go
-+++ b/pkg/apis/kubeflow/v2beta1/doc.go
-@@ -17,5 +17,5 @@
- // +k8s:openapi-gen=true
- 
- // Package v2beta1 is the v2beta1 version of the API.
--// +groupName=kubeflow.org
-+// +groupName=cw.xyz
- package v2beta1
-diff --git a/pkg/apis/kubeflow/v2beta1/register.go b/pkg/apis/kubeflow/v2beta1/register.go
-index fc14b70..f9ae7bd 100644
---- a/pkg/apis/kubeflow/v2beta1/register.go
-+++ b/pkg/apis/kubeflow/v2beta1/register.go
-@@ -21,10 +21,10 @@ import (
- )
- 
- const (
--	// GroupName is the group name use in this package.
--	GroupName = "kubeflow.org"
--	// Kind is the kind name.
--	Kind = "MPIJob"
-+       // GroupName is the group name use in this package.
-+       GroupName = "cw.xyz"
-+       // Kind is the kind name.
-+       Kind = "GroupJob"
- 	// GroupVersion is the version.
- 	GroupVersion = "v2beta1"
- )

index 5d710dd..770f663 100644
--- a/README.md
+++ b/README.md
@@ -1,9 +1,9 @@
-# MPI Operator
+# Group Operator
 
 [![Build Status](https://github.com/kubeflow/mpi-operator/workflows/build/badge.svg)](https://github.com/kubeflow/mpi-operator/actions?query=event%3Apush+branch%3Amaster)
 [![Docker Pulls](https://img.shields.io/docker/pulls/mpioperator/mpi-operator)](https://hub.docker.com/r/mpioperator/mpi-operator)
 
-The MPI Operator makes it easy to run allreduce-style distributed training on Kubernetes. Please check out [this blog post](https://medium.com/kubeflow/introduction-to-kubeflow-mpi-operator-and-industry-adoption-296d5f2e6edc) for an introduction to MPI Operator and its industry adoption.
+The Group Operator makes it easy to run allreduce-style distributed training on Kubernetes. Please check out [this blog post](https://medium.com/kubeflow/introduction-to-kubeflow-mpi-operator-and-industry-adoption-296d5f2e6edc) for an introduction to Group Operator and its industry adoption.
 
 ## Installation
 
diff --git a/deploy/v2beta1/mpi-operator.yaml b/deploy/v2beta1/mpi-operator.yaml
index caa0651..107461c 100644
--- a/deploy/v2beta1/mpi-operator.yaml
+++ b/deploy/v2beta1/mpi-operator.yaml
@@ -27,14 +27,14 @@ metadata:
     app.kubernetes.io/component: mpijob
     app.kubernetes.io/name: mpi-operator
     kustomize.component: mpi-operator
-  name: mpijobs.kubeflow.org
+  name: groupjobs.cw.xyz
 spec:
-  group: kubeflow.org
+  group: cw.xyz
   names:
-    kind: MPIJob
-    listKind: MPIJobList
-    plural: mpijobs
-    singular: mpijob
+    kind: GroupJob
+    listKind: GroupJobList
+    plural: groupjobs
+    singular: groupjob
   scope: Namespaced
   versions:
   - name: v2beta1
diff --git a/pkg/apis/kubeflow/v2beta1/doc.go b/pkg/apis/kubeflow/v2beta1/doc.go
index 45eb00b..25ce134 100644
--- a/pkg/apis/kubeflow/v2beta1/doc.go
+++ b/pkg/apis/kubeflow/v2beta1/doc.go
@@ -17,5 +17,5 @@
 // +k8s:openapi-gen=true
 
 // Package v2beta1 is the v2beta1 version of the API.
-// +groupName=kubeflow.org
+// +groupName=cw.xyz
 package v2beta1
diff --git a/pkg/apis/kubeflow/v2beta1/register.go b/pkg/apis/kubeflow/v2beta1/register.go
index fc14b70..f9ae7bd 100644
--- a/pkg/apis/kubeflow/v2beta1/register.go
+++ b/pkg/apis/kubeflow/v2beta1/register.go
@@ -21,10 +21,10 @@ import (
 )
 
 const (
-	// GroupName is the group name use in this package.
-	GroupName = "kubeflow.org"
-	// Kind is the kind name.
-	Kind = "MPIJob"
+       // GroupName is the group name use in this package.
+       GroupName = "cw.xyz"
+       // Kind is the kind name.
+       Kind = "GroupJob"
 	// GroupVersion is the version.
 	GroupVersion = "v2beta1"
 )
